cascades-server/persistence-hibernate/src/test/java/pl/gov/coi/cascades/server/persistance/hibernate/DatabaseInstanceGatewayImplTest.java

```diff
@@ -0,0 +1,70 @@
+package pl.gov.coi.cascades.server.persistance.hibernate;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.runners.MockitoJUnitRunner;
+import pl.gov.coi.cascades.contract.domain.DatabaseId;
+import pl.gov.coi.cascades.contract.domain.DatabaseType;
+import pl.gov.coi.cascades.contract.domain.NetworkBind;
+import pl.gov.coi.cascades.contract.domain.UsernameAndPasswordCredentials;
+import pl.gov.coi.cascades.server.domain.DatabaseInstance;
+import pl.gov.coi.cascades.server.domain.DatabaseStatus;
+import pl.gov.coi.cascades.server.persistance.stub.DatabaseTypeStub;
+import pl.gov.coi.cascades.server.persistance.stub.NetworkBindStub;
+import pl.gov.coi.cascades.server.persistance.stub.TemplateIdGatewayStub;
+import pl.gov.coi.cascades.server.persistance.stub.UsernameAndPasswordCredentialsStub;
+
+import java.time.Instant;
+import java.util.Date;
+
+import static junit.framework.TestCase.assertNull;
+
+/**
+ * @author Łukasz Małek <lukasz.malek@coi.gov.pl>
+ */
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
In this test you don't need mockito. BTW look how we setup mockito in this project.